python/ray/experimental/state.py

```diff
@@ -304,3 +303,20 @@ def client_table(self):
       node_info[node_ip_address].append(client_info_parsed)
 
     return node_info
+
+  def task_profiles(self):
+    """Fetch and return a list of task profiles.
+
+    Returns:
+      A list of task profiles.
+    """
+    r = ray.worker.global_worker.redis_client
+    event_names = r.keys("event_log*")
+    results = []
+    for i in range(len(event_names)):
+      event_list = r.lrange(event_names[i], 0, -1)
+      decoded = []
```

Comment:
I wonder if this should be a list or if it should be a dictionary mapping task ID to info